AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginAuthIAMTests/AWSDataStoreAuthBaseTest.swift

```diff
@@ -203,57 +197,67 @@ extension AWSDataStoreAuthBaseTest {
     /// - Parameter user
     func signIn(user: TestUser?,
                 file: StaticString = #file,
-                line: UInt = #line) {
+                line: UInt = #line) async {
```

Comment:
isn't it `async throws` ?

Reply:
There's a `do { } catch { }` statement  at line 206 which catches the thrown error and fails the test